assistants/core/src/index.ts

```diff
@@ -48,7 +48,7 @@ import zhHansMessages from './locale/zh-Hans/messages'
 
 export type CreateRuleFunction = (i18n: I18n) => RuleDefinition
 
-const SUPPORTED_LOCALES = ['en', 'zh-Hans']
+const SUPPORTED_LOCALES = ['en', 'zh-Hans', 'zh-CN']
```

Comment:
I wonder why _Chinese (China)_ got added? Sketch only supports _English_ and _Chinese (Simplified)_, i.e. `zh-Hans`. However I know that Crowdin by default uses `zh-CN` and therefor the project custom language setting, see [their documentation](https://support.crowdin.com/changing-target-languages/#custom-languages).

Reply:
Ah I just added this while testing the localisation in passing. Should have been discussed in a separate issue.

If you set your region as _Chinese Simplified_ in macOS, then the locale reported by the OS (`defaults read -globalDomain AppleLocale`) is `zh-CN`, not `zh-Hans`. As I understand it `zh-Hans` is the more modern/correct value, but many systems still use `zh-CN`.

So this isn't technically adding `zh-CN` as new locale to localise for (that is defined by the po files), rather it's supporting `zh-CN` as an alias for `zh-Hans`:

```
catalogs: {
  en: enMessages,
  'zh-Hans': zhHansMessages,
  'zh-CN': zhHansMessages,
},
```

Can remove if you like.